### ISSUE TITLE
Unable to reproduce results of Nuscenes free-anchor

### ISSUE THREAD

**Issue by Lucy0601** [open-mmlab/mmdetection3d#64]
For the config file: hv_pointpillars_fpn_sbn-all_free-anchor_4x8_2x_nus-3d.py
The report result is: mAP-43.7 NDS-55.3
However our reproduce result is only: mAP-14.9 NDS-27.6
The only change is (because we don't have the sufficient memory)
samples_per_gpu=4
->
samples_per_gpu=2
So, does the small batch_size will decrease the accuracy?

**Comment by Tai-Wang** [comment]
Please follow the template for "error report" to describe your issue.

A small batch size may affect the accuracy slightly (maybe about 1-2 mAP). From your current description, your case is more like a bug. You can check the data preparation procedure and try other configs (for different models on different datasets) for more cues.

**Comment by ZwwWayne** [comment]
Small batch_size will decrease accuracy but should not decrease the AP by more than 10 points. If other things are OK and you only changed the batch size, the performance could drop due to the following reasons:
1. The learning rate is a hyperparameter that should be modified according to the overall batch size. You need to tune a smaller learning rate that is suitable for batch size 2x8.
2. The nuScenes dataset is an imbalanced dataset, so the batch size will also affect the performance in this part.

With a sufficient hyperparameter tuning for bs=2x8, you could expect results that have 1~2 AP and NDS lower than that of the released setting.

**Comment by Lucy0601** [comment]
Thank you, 
the other configs of Nuscenes have the right accuracy, 
I will try a small learning rate to validate the result.

**Comment by ZwwWayne** [comment]
If other configs are fine, the codebase should be OK. We have not tried FreeAnchor with smaller batch size and only tried this on the PointPillar yet, so it is also possible the LR & batch size have more impact on the results with FreeAnchor.
